src/pages/innovation-day/index.mdx

```diff
@@ -0,0 +1,82 @@
+description: 'Register for wasmCloud Innovation Day 2024'
+---
+
+![wasmCloud Innovation Day](../../../static/pages/home/innovation-day/innovation-day.png)
+
+
+**Time:** Wednesday, September 18, 2024 @ 10am - 6pm Eastern  
+**Place:** Virtual
+
+Whether you’re already using Wasm or a complete newbie, this one-day virtual event is the ideal immersion in all things WebAssembly and wasmCloud. We’ll explore wasmCloud’s technical architecture and growing feature set&mdash;and most importantly, hear how companies in a host of industries are working with this popular open source CNCF project.
+
+Here’s what to expect…check out the full agenda below!
+
+* **wasmCloud in Industry:** Hear how engineers from the world’s largest organizations are using wasmCloud in real-world use cases.
+
+* **Technical Deep Dive:** wasmCloud has been transformed with WASI 0.2 and wasmCloud 1.1 is full of features designed to make building with components a joy. From secrets support to creating your own custom capabilities, we’ll take a deeper look at what’s new, and how to get the most out of 1.1.
+
+* **Community Workshop and Hangout:** Join us in Gather space to build your own Wasm application and hang with industry leaders, open source contributors and wasmCloud users.
+
+<div style={{ display: 'table', width: `1200px`, paddingTop: '20px' }}>
+    <div style={{ float: 'left', width: '40%' }}>
+        <h2>Agenda</h2>
+
+**Morning Session** 
+
+**10 - 11 ET: wasmCloud Weekly Community Meeting**
+
+- Special edition: we’ll focus on the essential role of open source maintainers: explore what being a maintainer involves, and how you can work with us.
+
+**11 - 11.15: Opening Keynote:** **The Pitfalls of Modern Platform Engineering: [Bailey Hayes](https://github.com/ricochet)**
+
+- Ending the ‘rinse and repeat’ of application development
+- Improving life with Kubernetes
+- Running distributed workloads securely and efficiently
+
+**11.15 - 12.15: wasmCloud 1:1, Technical Deep Dive**
+
+- **Secrets: [Brooks Townsend](https://github.com/brooksmtownsend)**
+    - Why this is the most requested feature amongst our enterprise customers and partners
+- **Evolution in Golang: [Lucas Fontes](https://github.com/lxfontes), [Brooks Townsend](https://github.com/brooksmtownsend)**
+    - Go provider SDK; Go components with TinyGo
+- **wasmcloud-operator: Lucas Fontes**
+- **wash dev: [Taylor Thomas](https://github.com/thomastaylor312)**
+- **Custom Capabilities**
+    - Postgres and Couchbase with [Victor Adossi](https://github.com/t3hmrman); Machine Learning with Christoph Brewing, BMW / Jochen Rau, MachineMetrics
+- Seamless distributed systems with **wRPC and NATS:** [Taylor Thomas](https://github.com/thomastaylor312) and [Roman Volosatvos](https://github.com/rvolosatovs)
+
+**12.15 - 1.30pm: Community Keynote and Demos**
+
+- **12.15 - 12.30: Fireside Chat:** Community leaders in the CNCF and Bytecode Alliance join [Bailey Hayes](https://github.com/ricochet)
```

Comment:
Demote this and add timing to community demo:

12.15 - 12.30pm: Fireside chat
bullet

12.30 - 1.30pm: Customer and user demos
bullet You don't xxx